methods/lisk.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "lisk",
+  "status": "registered",
+  "verifiableDataRegistry": "Lisk Sidechain",
+  "contactName": "Aldo Suhartono Putra",
+  "contactEmail": "",
```

```suggestion
  "contactEmail": "aldhosutra@gmail.com",
```

Please raise another PR to add your contact information (which is contained in the DID Method spec you reference). Otherwise, we might not be able to contact you if the registration requirements change and/or we need to confirm that you DID Method is still valid.